lib/akamai_rspec/request.rb

```diff
@@ -69,7 +69,12 @@ def self.responsify(maybe_a_url)
       if maybe_a_url.is_a? RestClient::Response
         maybe_a_url
       else
-        RestClient.get(maybe_a_url, akamai_debug_headers)
+        begin
+          RestClient.get(maybe_a_url, akamai_debug_headers)
+        rescue RestClient::RequestFailed => exception
+          # Return the original request
+          exception.response
```

Comment:
When you get a forbidden (403) or any other 4/5xx response rest-client raises an exception.

Sometimes that is expected and we want to test for it, as well as check the headers of the returned request.

Yes, exception.response is the original response as per http://www.rubydoc.info/gems/rest-client/1.6.1/RestClient/Exception#response-instance_method